### DISCUSSION TITLE
Can we adjust the FOV?

### DISCUSSION THREAD

**Discussion by casedistorted** [sultim-t/xash-rt#106] [Q&A]
So I thought I remember the console command being default_fov 90, but for some reason there's no option to enable developer console in-game pressing the tilde ~ key.  I checked in all the options available but nothing so far.

if there's a .cfg or .ini file we could edit to fix this that's fine, but the default fov of 90 on widescreen is too zoomed in for me to not feel motion sickness.  If I remember correctly the original game was 90 fov in 4:3, which worked, but when stretched to 16:9 widescreen the default fov of 90 is too zoomed in.

Also I changed my resolution to x2.25 my original 1440p with DSR with Nvidia Control Panel and because the game is in windowed mode (no option for fullscreen?) the window stretched across both of my monitors and I could not click on graphics again to switch it back to 1440p.  Somehow managed to bring it back to "windowed fullscreen" by using some windows keyboard shortcuts but a little finnicky and almost locked me out of the game.

**Comment by sultim-t** [comment]
In the latest build, there should be `rt_bin/xash3d-console.bat` to launch with a console

**Comment by Nord1cWarr1or** [comment]
> So I thought I remember the console command being default_fov 90, but for some reason there's no option to enable developer console in-game pressing the tilde ~ key. I checked in all the options available but nothing so far.

Just add `-console` in launch parameters
